templates/amqp1.conf.j2

```diff
@@ -1,14 +1,22 @@
 #TODO: Add new value for Host and port
 #TODO: create a loop to parse input and generate the Instance members
 
+{% if plugin_amqp1_interval is not defined %}
 LoadPlugin amqp1
+{% else %}
+<LoadPlugin amqp1>
+   Interval {{ collectd_plugin_amqp1_interval }}
```

Comment:
Interval belongs to the plugin section?

Reply:
That's what I thought initially, however, all the sample configs show Interval in the LoadPlugin section.

It's documented under the `Global Options` section and is an option for `LoadPlugin`: https://collectd.org/documentation/manpages/collectd.conf.5.shtml#global_options

Reply:
In my understanding, you can have the interval in the global section and also for every plugin.